server/Controllers/PointsController.cs

```diff
@@ -45,6 +50,7 @@ public PointsController(IMemoryCache memoryCache)
                         .SetPriority(CacheItemPriority.NeverRemove)
                 );
             }
+            _logger = logger;
         }
 
         // GET /api/points
```

Comment:
what do we do for next situation:
 1. tags.json contains tag "run"
 2. one point for "run" has been submitted, validated and stored to storage.json
 3. tags.json has been modified manually: "run" was replaced by "walk"
 4. one point for "walk" has been submitted, validated and stored to storage.json
 5. I GET all points from storage.json to a user buffer.
 6. I do PostAll same points from the user buffer
 7. I get validation exceptions.